### PR TITLE
pkp/pkp-lib#2523: Remove sections added by Native Import if error occurred

### PR DESCRIPTION
**Pull request by ctgraham**
Resolves pkp/pkp-lib#2523.

### CODE REVIEW

**Comment by ctgraham** [comment]
Added commit to also resolve pkp/pkp-lib#2526 and to ensure issue errors remain as non-fatal warnings.

File: plugins/importexport/native/NativeImportExportPlugin.inc.php [hunk 1]
```diff
@@ -136,30 +136,40 @@ function display($args, $request) {
...
-					if (!empty($issuesErrors)) {
-						$templateMgr->assign('issuesErrors', $processedIssuesIds);
+
+				// Are there any import errors? Display them.
+				$errorTypes = array(
+					ASSOC_TYPE_ISSUE => 'issuesErrors',
+					ASSOC_TYPE_SUBMISSION => 'submissionsErrors',
+					ASSOC_TYPE_SECTION => 'sectionErrors',
+				);
+				$foundErrors = array();
```
Comment (bozana) [inline plugins/importexport/native/NativeImportExportPlugin.inc.php]: Do we need this variable as array? -- Maybe a boolean would be enough?

**Comment by bozana** [comment]
Hmmm... I cannot find this pkp-lib branch: ulsdevteam:native-import-cleanup-failed-import-sections (from the submodule update commit)  :-\

File: plugins/importexport/native/NativeImportExportPlugin.inc.php (hunk 1 above)
Comment (ctgraham) [inline plugins/importexport/native/NativeImportExportPlugin.inc.php]: Agreed.

**Comment by ctgraham** [comment]
Yes, this should have been ulsdevteam:native-import-add-warnings.  Will fix.

**Pull request merged**